.github/workflows/ci-polaris-react.yml

```diff
@@ -87,6 +88,7 @@ jobs:
             ${{ runner.os }}-node${{ matrix.node-version }}-visual-regression-test-v1-
 
       - run: yarn --frozen-lockfile
+      - run: yarn build
```

Comment:
I don't like having to build the packages before Storybook. This is where smart build pipeline would come in handy.

Rather than building ALL packages, we should defined only the packages that need to be built prior to building other assets like storybook